src/views/Results.vue

```diff
@@ -73,19 +73,19 @@ const share = () => {
     <TabGroup>
       <TabList class="flex space-x-1 rounded-t-lg bg-primary-purple/30 p-1">
         <Tab as="template" v-slot="{ selected }">
-          <button class="tab-button" :class="{ selected: selected }">
```

Comment:
Seems like Headless UI already applies the correct role to the button (is this also the case for you?)

Reply:
That was indeed an oversight. I have reverted the commit.